nfts/frontend/src/Components/NFT/TransferNFT.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import TransferNFTService from "../../utils/transferNft";
+import "./TransferNFT.css";
+
+export const TransferNFT = ({ sdk }) => {
+  const [tokenId, setTokenId] = useState("");
+  const [to, setTo] = useState("");
+  const [from, setFrom] = useState("");
+  const [result, setResult] = useState(null);
+
+  let handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const result = await TransferNFTService(tokenId, to, from);
+      if (result.status === "USER_CONSENT_NEEDED") {
+        const consentToken = result.consentToken;
+        const consent = await sdk.getConsent(consentToken);
+        if (consent.status === "QUEUED") {
+          setResult(consent);
+          console.log(result);
+        } else {
+          alert(consent.status);
+        }
+      } else {
+        alert(result.status);
```

Comment:
alert isn't showing the correct thing

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/100057389/193894503-f1def26b-ea38-451c-8328-34803cfef8c8.png">

Reply:
fixed

Reply:
It still wan't showing the actual error message. I have updated it.